### PR TITLE
Package ip2locationio.1.0.0

### PR DESCRIPTION
**Pull request by ip2location**
### `ip2locationio.1.0.0`
IP2Location.io OCaml module to get geolocation and WHOIS data
IP2Location.io OCaml module allows user to query for an enriched data set based on IP address and provides WHOIS lookup api that helps users to obtain domain information.

---
* Homepage: https://github.com/ip2location/ip2location-io-ocaml
* Source repo: git+https://github.com/ip2location/ip2location-io-ocaml.git
* Bug tracker: https://github.com/ip2location/ip2location-io-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ip2locationio/ip2locationio.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+tags: ["ip2locationio" "geolocation"]
+homepage: "https://github.com/ip2location/ip2location-io-ocaml"
+doc: "https://github.com/ip2location/ip2location-io-ocaml"
+bug-reports: "https://github.com/ip2location/ip2location-io-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.4"}
+  "lwt"
+  "cohttp"
+  "cohttp-lwt-unix"
```
Comment (haochenx) [inline packages/ip2locationio/ip2locationio.1.0.0/opam:18]: You probably need the following to run the tests on the CI

```suggestion
  "cohttp-lwt-unix"
  "tls-lwt" {with-test}
```

(there is a choice between `tls-lwt` and `lwt_ssl` and I'm suggesting `tls-lwt` here for no particular reason)

**Comment by ip2location** [comment]
Not sure what else I'm missing here. Would appreciate any guidance on fixing these issues.

**Comment by mseri** [comment]
The error is misleading here. The problem is that it is not possible to use the network in the CI due to the opam sandbox. I suggest to disable the tests altogether

File: packages/ip2locationio/ip2locationio.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
Comment (mseri) [inline packages/ip2locationio/ip2locationio.1.0.0/opam:34]:
```suggestion
```

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**